package.json

```diff
@@ -22,7 +22,7 @@
     "url": "http://github.com/home-assistant/homebridge-homeassistant/issues"
   },
   "engines": {
-    "node": ">=0.12.0",
+    "node": ">=6.0.0",
```

Comment:
I'm at node -v 4.3.7 but getting errors when running your modifications... Probably something simple I've overlooked and any help appreciated:

`SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mod`